http/baker-http-server/src/main/scala/com/ing/baker/http/server/javadsl/BakerWithHttpResponse.scala

```diff
@@ -87,6 +87,41 @@ class BakerWithHttpResponse(val baker: Baker, ec: ExecutionContext) extends Lazy
     }
   }
 
+  def parseEventRequest(eventJson: String): Either[BakerException, EventInstance] = {
+    parse(eventJson) match {
+      case Left(_) =>
+        logger.error("Failure parsing of EventInstance")
+        Left(BakerException.UnexpectedException("Failure parsing of EventInstance"))
+      case Right(json: io.circe.Json) =>
+        json.as[EventInstance] match {
+          case Left(_) =>
+            logger.error("Failure parsing of EventInstance")
+            Left(BakerException.UnexpectedException("Failure parsing of EventInstance"))
```

Comment:
Might be better to give a distinct log message and exception, might save some headache later looking at the wrong place

Reply:
I have kept this one but changed the second error message.